client/src/pages/BookingPage/BookingPage.tsx

```diff
@@ -2,23 +2,72 @@ import Box from '@mui/material/Box';
 import Grid from '@mui/material/Grid';
 import BookingWrapper from '../../components/ManageBookings/BookingWrapper/BookingWrapper';
 import CurrentBookingCard from '../../components/ManageBookings/CurrentBooking/CurrentBookingCard';
-import BookingCalendar from '../../components/ManageBookings/BookingCalendar/BookingCalendar';
 import BookingCalendarWrapper from '../../components/ManageBookings/BookingCalenderWrapper/BookingCalenderWrapper';
 import useStyles from './useStyles';
+import { useState, useEffect } from 'react';
+import { fetchAllRequest } from '../../helpers/APICalls/requests';
+import { Request } from '../../interface/manageBooking';
+
+const obj: Request = {
+  _id: 'djfkdjf',
+  start: '02/28/2022',
+  status: 'Not Available',
+  user: {
+    photo: '',
+    name: 'NoBookings',
+  },
+};
+
+interface requestObj {
+  accepted: Request[];
+  pending: Request[];
+  cancelled: Request[];
+}
```

Comment:
Can't we utilize the interfaces created in the interfaces folder rather than redefining them here?